### PR TITLE
Update documentation/content/en/books/arch-handbook/boot/_index.adoc

### PR DESCRIPTION
**Pull request by isoux**
### **documentation/content/en/books/arch-handbook/boot/**
A lot has changed in the code since 2014 and RELEASE 10. The most significant change is in the path to the boot folder:  /usr/src/sys/boot -> /usr/src/stand/

### CODE REVIEW

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (pauamma) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:908]: I'm not sure I'd drag metamode in here - build methods have nothing to do with the boor process IMO. Even if you want metamode (here or elsewhere), I'd link to https://wiki.freebsd.org/MetaMode.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (pauamma) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:968]: s/functions/function/ while here.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (pauamma) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:1327]: For consistency with earlier language referencing functions, I'd either remove "The" or change it to "The function".

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:908]: You are totally right, I just wanted to show how I came to the link command because it is now excluded from the mentioned file. Are you suggesting that I totally remove the mention of METAMOE?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:968]: Yes, mistake, I'll fix that.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:1327]: Thanks, I'll fix it.

**Comment by isoux** [comment]
Thank you very much!
Since this is my first pull request, which I need to correct, do I need to correct errors on my repository and then do a pull request again, or is there another way?

**Comment by isoux** [comment]
Sorry, the last request was an accidental mistake due to insufficient knowledge of git. Please delete it.

**Comment by pauamma** [comment]
English-wise, LGTM. I'll let others look at accuracy.

**Review by debdrup — CHANGES_REQUESTED** [review]
All the paths in the old version are relative to /usr/src.

Is there a reason for changing this?

If not, I'd prefer if we keep it the same.

**Comment by isoux** [comment]
Thank you very much!
> All the paths in the old version are relative to /usr/src.
> 
> Is there a reason for changing this?
> 
> If not, I'd prefer if we keep it the same.

Do you associate this with:
For example, if before was:
_sys/ boot/i386/boot/boot0.S_
It should now be:
_stand/i386/boot0/boot0.S (without usr/src/)_?

Then you're right it shouldn't change, I'll fix it.

> In addition to this, it looks like some of the spacing (from tabs and spaces) has been changed, and I don't think that's a good idea either.

When I added new lines of code, in accordance with the changes in the code so far, I tried to arrange the comments in the same plane so that they do not stick out here and there ... In my editor, it now looks sorted out. If it looks messy on the web page, I'll do my best to return it to the original?

**Comment by isoux** [comment]
Sorry, another wrong pull request by mistake...

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:505]: There are also some tabs here that need to be realigned.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:289]: There's a tab that gets removed here, which I don't think is what's intended as it moves the comment.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:568]: There's a full path used in this this.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:60]: There's a full path used in this this.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:646]: There's a full path used in this this.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:1050]: Where are you getting this code from?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:968]: This is not correct.

**Review by debdrup — CHANGES_REQUESTED** [review]
I'm gonna need to go over this with a fine comb tooth, because there's a lot of things that don't make sense to me.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:568]: I have left it as it is in the original document full path used in this one.It Isn't hard for me to fix it if I have to?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:60]: I have left it as it is in the original document full path used in this one.It Isn't hard for me to fix it if I have to?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:646]: I have left it as it is in the original document full path used in this one.It Isn't hard for me to fix it if I have to?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:505]: Thank you for your patience, I apologize for the beginner's clumsiness!

I'll fix it.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:289]: I'll fix it.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:1050]: From sys/kern/subr_param.c
Maybe you're suggesting I delete the first line? (hz = -1;)

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:505]: ... I tried to correct it... I'm really confused because of the lack of knowledge, because of the language barrier, and because of the fact that I can't in any other way equalize these comments to be on the same level. If I try it as you suggested, then they stick out here and there. If it suits you, I will do it, but in the original, they are arranged. It's not my fault that the code has changed, so some lines have become too long. If you look at these comments on my repository now, everything is fine.

Is it more important for us to strictly keep the positions of the tabs or a nice look of aligned comments as in the original?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:1050]: Nope, sorry. I was just confused because it's such a big change.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:968]: I think I'd prefer getting a source committer involved in that, because I'm "just" a doc committer.

I'll try to find someone. :)

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:505]: Maybe it's that what I'm seeing in the github preview isn't looking like it looks in a terminal?

Anyway, the important thing is that it's as accurate to the current code and indentation as possible.

**Review by debdrup — COMMENTED** [review]
I really appreciate you taking the time to work with this back and forth  :)

As I said in the review, I'll find a source committer to look it over too.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (bsdimp) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:60]: "MBR" isn't the standard. It's called "CSM boot" these days. It happens to use MBR to implement it, though.
GPT also is a partitioning standard, not a booting standard. And ZFS isn't a boot standard, but a filesystem.
EFI is more covnetionally referred to UEFI since the EFI standard is 20 years obsolete.

So we support CSM (with both GPT and MBR partitioning support) and UEFI booting (GPT is totally supported, MBR is mostly supported). We support loading files from ext2fs, MSDOS, UFS and ZFS. We also support the boot environment feature of ZFS which allows the HOST OS (FreeBSD) to communicate details about what to boot that go beyond a simple partition which was possible in the past. 

I get that it may be hard to convey these nuances, but the above paragraph needs additional work since it's written with the older sensibilities and reads as somewhat dated.

Finally, the docs are moving to 'One sentence per line' rather than 'One paragraph per line' style...

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (bsdimp) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:146]: This reads funny. We're calling the BIOS to read the MBR and this is a rather indirect way of saying so.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (bsdimp) [inline documentation/content/en/books/arch-handbook/boot/_index.adoc:908]: Since you are adding content here, I'd do new sentences per line (unlike above where it's pre-existing structure and you're just changing relatively minor details)

**Comment by bsdimp** [comment]
Generally speaking, I like these changes. The code is mostly right (I didn't concentrate enough to be more definitive about that), but I'd suggest as a followup that amd64 and arm64 are much more important and compelling examples. And UEFI is more relevant than the CMS discussions.

**Comment by isoux** [comment]
Thanks a lot to **both of You**! 
I am very happy "to have this sweet torment around the effort to improve this document". As soon as I have time, I will return to correcting and replying to your messages.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (debdrup) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:908]: I plan on doing a pass of one sentence per line before I push this into the docs tree, after it's been approved.

Comment (bsdimp) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:908]: OK. Once you are happy with the formatting, I'm happy (enough) with the content. It updates what's there to be correct, so that's progress.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:505]: Here's what it looks like in my github preview. I do not use a correction terminal

Git_set
.
So should I change that or leave it that way?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:289]: I fixed this.

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:568]: So should I change that or leave it that way?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:60]: So this is a suggestion on how to implement your proposal: 

About 'One sentence per line' rather than 'One paragraph per line' style, I would leave the @debdrup to tidy up as promised?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:1050]: Thank you!

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:146]: Maybe an expression like this is a better solution for now? :

**As mentioned previously, the called bios interrupt causes the `INT 0x19` handler to load an MBR** ... ?
or
**As mentioned previously, the called BIOS interrupt `INT 0x19` causes the handler to load an MBR** ... ?

File: documentation/content/en/books/arch-handbook/boot/_index.adoc
Comment (isoux) [reply documentation/content/en/books/arch-handbook/boot/_index.adoc:908]: **Thank you very much!** I am very happy!
I am waiting for your final answers and suggestions and I am approaching the final correction.

**Comment by isoux** [comment]
I corrected everything as @debdrup requested and @bsdimp suggested.

There seems to be something wrong with the tabs again ... now it's clearer to me how to fix it, so let me know if it's wrong for me to fix it.

**Review by pauamma — CHANGES_REQUESTED** [review]
Some grammar and spelling fixes, but LGTM otherwise. Almost there.

**Comment by pauamma** [comment]
I'd go for "Reviewed-by", not "Co-authored-by". I don't think I contributed nearly enough content for that. LGTM now, but I'll let debdrup pipe in if wanted.

**Review by debdrup — APPROVED** [review]
I'll accept this tentatively now, and get to work on doing a final pass  plus fixing one sentence per line.

I will need one more thing from you before being able to commit though; in order to provide proper credit, I'll need the following:
Givenname Familyname <user@email.tld>

This will be used as for git commit --author.

EDIT: This is no longer needed, I found out it's included in the patch I get from github. :)

**Comment by isoux** [comment]
Thanks, man and excuse me for my clumsiness.

**Comment by debdrup** [comment]
Sorry, I was intending to do this, but had to be added to the triage team here on github first, but you closed it before I noticed I'd been added.

Again, thank you for your contribution - and hopefully you can find something else to contribute? ;)

**Comment by isoux** [comment]
You don't have to worry or be sorry.

I saw that this was not closed and I thought I should close it because I saw that the changes had already been updated on the document's website.

I am overjoyed because I dared to embark on something that is beyond my capabilities ... I am sorry that I bothered you with my clumsiness, surely the next contribution will be less painless for you and me :)

Thank you for accepting this responsible and unenviable job.

**Pull request closed without merging**
